playground.html

```diff
@@ -350,9 +350,12 @@
                     <wb-value type="array">item from</wb-value>
                     <wb-value type="number" value="0" min="0">at index</wb-value>
                 </wb-expression>
-                <wb-expression type="string" ns="array" fn="join">
+                <wb-expression type="array" ns="array" fn="join">
```

Comment:
Maybe we should name this "concatenate" or "join arrays" to keep people who are familiar with JS from confusing with the string join in JS arrays?